content/docs/06-integrations/00-antrea-cni.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Antrea CNI'
+metaTitle: 'Antrea CNI'
+metaDescription: 'Antrea CNI network pack for Palette Kubernetes Clusters'
+hiddenFromNav: true
+type: "integration"
+category: ['network']
+logoUrl: 'https://registry.spectrocloud.com/v1/palette-upgrader/blobs/sha256:b6081bca439eeb01a8d43b3cb6895df4c088f80af978856ddc0da568e5c09365?type=image/png'
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Antrea CNI
+
+Palette supports Antrea controller network interface (CNI) for Kubernetes clusters. Antrea  CNI enables each pod to have exclusive IP addresses from the subnet with direct accessibility. 
+
+Antrea leverages [Open vSwitch](https://www.openvswitch.org/) to implement pod networking and security features. Open vSwitch enables Antrea to implement Kubernetes network policies efficiently.
+
+# Supported Versions
+
+**1.9.x**
+
+# Prerequisites
+
+- Enable the ``enableNodeIPAM`` integrated NodeIPAM controller within the Antrea controller.
+- When deploying a cluster using kubeadm, specify the ``--pod-network-cidr <cidr>`` option and provide the IP address with the classless inter-domain routing (CIDR). For example: 
+
+    ``--pod-network-cidr=10.244.0.0/16``
+
+    <br />
+
+    <WarningBox>
+
+    To avoid overlapping your pod network with any of your host networks, you should think of a suitable CIDR block to specify if you deploy a cluster using ``kubeadm init`` with ``--pod-network-cidr <cidr>`` or as a replacement in your network plugin's YAML.
```

Comment:
For palette, podCIDR mentioned in the k8s values.yaml will always take preference. If you want to use the Antrea CIDRs, (only podCIDR works, service CIDR doesn’t) you need to empty values for pod and service CIDR in the k8s pack.

Reply:
@roshanirathi Please review Troubleshooting section. You can see the doc at the Preview link in the PR Description.